tests/main/install-snaps/task.yaml

```diff
@@ -0,0 +1,82 @@
+summary: Check install popular snaps
+
+manual: true
+
+environment:
+    # High Profile
+    SNAP/azurecli: azure-cli
+    SNAP/awscli: aws-cli
+    SNAP/heroku: heroku
+    SNAP/hiri: hiri
+    SNAP/kubectl: kubectl
+    SNAP/rocketchatserver: rocketchat-server
+    # Selected from recent insights posts
+    SNAP/corebird: corebird
+    SNAP/gitterdesktop: gitter-desktop
+    SNAP/helm: helm
+    SNAP/mattermostdesktop: mattermost-desktop
+    SNAP/mentaplexmediaserver: menta-plexmediaserver
+    SNAP/openspades: openspades
+    SNAP/pintown: pin-town
+    SNAP/postgresql10: postgresql10
+    SNAP/storjshare: storjshare
+    SNAP/slackterm: slack-term
+    SNAP/vectr: vectr
+    SNAP/wekan: wekan
+    SNAP/wormhole: wormhole
+    # Featured snaps in Ubuntu Software
+    SNAP/anboxinstaller: anbox-installer
+    SNAP/lxd: lxd
+    # Top non canonical snaps
+    SNAP/atom: atom
+    SNAP/discord: discord
+    SNAP/docker: docker
+    SNAP/etcd: etcd
+    SNAP/geocoder: geocoder
+    SNAP/gimp: gimp
+    SNAP/huggle: huggle
+    SNAP/hugo: hugo
+    SNAP/ia: ia
+    SNAP/kurly: kurly
+    SNAP/micro: micro
+    SNAP/nikola: nikola
+    SNAP/parity: parity
+    SNAP/paritybitcoin: parity-bitcoin
+    SNAP/remmina: remmina
+    SNAP/telegramsergiusens: telegram-sergiusens
+    SNAP/zeronet: zeronet
+    SNAP/zeronetjs: zeronet-js
+    # Top canonical snaps
+    SNAP/bare: bare
+    SNAP/bluez: bluez
+    SNAP/conjureup: conjure-up
+    SNAP/gedit: gedit
+    SNAP/go: go
+    SNAP/juju: juju
+    SNAP/neutron: neutron
+    SNAP/nova: nova
+    SNAP/snapcraft: snapcraft
+    SNAP/solc: solc
+    SNAP/vault: vault
+
+execute: |
+    CHANNELS=(stable candidate beta edge)
```

Comment:
(nitpick, not really important as we use bash in spread anyway but I can't help it, years of ensuring maintainer scripts work with dash (instead of bash) have tained me :)) - if you want to write this in portable "sh" instead of bash you could simpliy write it as.
```
CHANNELS="stable candidate beta edge"
for CHANNEL in $CHANNELS; do
...
```